infrastructure/models.py

```diff
@@ -308,6 +337,54 @@ def fuzzy_output_date(self):
     estimated_transfer_capacity = models.BigIntegerField(
         null=True, blank=True, help_text="Estimated Transfer Capacity (MW)"
     )
+    # ---- END TRANSMISSION TYPE PROJECT FIELDS ---- #
+
+    # ---- PIPELINE TYPE PROJECT FIELDS ---- #
+    pipeline_diameter = models.IntegerField("Pipeline Diameter", null=True, blank=True)
+    pipeline_diameter_unit = models.CharField("Diameter Unit", null=True, blank=True, max_length=6,
+                                              choices=(
+                                                  ('inches', 'inches'),
```

Comment:
Best practice is to [define a separate choices variable inside a model class](https://docs.djangoproject.com/en/2.1/ref/models/fields/#django.db.models.Field.choices). RA does it slightly differently though, so I'd try to follow form with the existing code base first, like ``ProjectStatus`` [here](https://github.com/CSIS-iLab/new-silk-road/blob/staging/infrastructure/models.py#L105-L114).

Reply:
Oh, I see choices was used for the other fields, but not these, was that intentional?

Reply:
I had seen it on transmission, and since it was a small list that impacted only this one field I figured it was okay. I'll move it into a model. Should I move the `electricity_flow` into a new model as well?